apps/src/music/utils/Loader.ts

```diff
@@ -1,7 +1,10 @@
 // Utilities for retrieving various types of data for Music Lab.
 
-import {getJson} from '@cdo/apps/util/HttpClient';
-import MusicLibrary, {LibraryJson} from '../player/MusicLibrary';
+import {getJson, ResponseValidator} from '@cdo/apps/util/HttpClient';
```

Comment:
Curious whether `getJson` is a little generic, and maybe `fetchJson` or even a call like `HttpClient.getJson` might be a bit more recognizable in code that uses it?

Reply:
+1, `getJson` could be interpreted as not actually making an http request